### ISSUE TITLE
Where is FreeLLama?

### ISSUE THREAD

**Issue by volkanorhan** [WinampDesktop/winamp#64]
A few months ago (on May 16, 2024), Winamp announced that Winamp will be open source with the name of FreeLLama. It is very sad not to see Winamp completely free and open source. Some components of Winamp such as MAKI Compiler are not available with source codes. I would want to see Winamp completely free and open source without any restriction. Please make Winamp completely open source and completely free (as in free speech) by changing its licensing model and open sourcing all of its components (like MAKI Compiler).
The related links are below:
https://about.winamp.com/free-llama
https://about.winamp.com/press/article/winamp-open-source-code
https://linuxiac.com/winamp-becomes-open-source-under-the-freellama-name/

**Comment by jakubtalich** [comment]
Google:
Winamp is not going open source. Here's what it is doing - ZDNET https://www.zdnet.com/home-and-office/home-entertainment/winamp-is-not-going-open-source-heres-what-it-is-doing-and-why/
"The sort-of open-source version is going by the name FreeLLama. While Winamp hasn't said yet what license it will use for this forthcoming version, it cannot be open source."

**Comment by volkanorhan** [comment]
> Google: Winamp is not going open source. Here's what it is doing - ZDNET https://www.zdnet.com/home-and-office/home-entertainment/winamp-is-not-going-open-source-heres-what-it-is-doing-and-why/ "The sort-of open-source version is going by the name FreeLLama. While Winamp hasn't said yet what license it will use for this forthcoming version, it cannot be open source."

I would want to see Winamp as completely free and completely open source.

**Comment by volkanorhan** [comment]
Yes, Winamp should free the LLama by releasing a proper license.
